src/Catalog/Persistence/AzureStorage.cs

```diff
@@ -339,13 +340,15 @@ protected override async Task<StorageContent> OnLoadAsync(Uri resourceUri, Cance
 
                 return new StringStorageContent(content);
             }
-
-            if (Verbose)
+            catch (StorageException ex) when (ex.RequestInformation.HttpStatusCode == (int)HttpStatusCode.NotFound)
```

Comment:
I believe we use `.RequestInformation?.HttpStatusCode` in other areas, do we need to follow that pattern here?

Reply:
Hmm... that is a good point. I did not consider this being null. I think I investigated this in the past by digging through storage SDK code but I forget the conclusion. Best to be defensive in the `catch` case.